src/contracts/BCoWFactory.sol

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity 0.8.25;
+
+import {BCoWPool} from './BCoWPool.sol';
+
+import {BFactory} from './BFactory.sol';
+import {IBFactory} from 'interfaces/IBFactory.sol';
+import {IBPool} from 'interfaces/IBPool.sol';
+
+/**
+ * @title BCoWFactory
+ * @notice Creates new BPools, logging their addresses and acting as a registry of pools.
+ */
+contract BCoWFactory is BFactory {
+  address public solutionSettler;
+
+  constructor(address _solutionSettler) BFactory() {
+    solutionSettler = _solutionSettler;
+  }
```

Comment:
i'd make it immutable for the sake of not adding a setter (and complexity) to the design
`bLabs` is only allowed to `collect`, but doesn't have config powers, if the solution settler changes, then all current pools are already binded to the old one, and for new pools we'll need to deploy a new factory
most probable outcome if it changes, we'd require to do some changes